### DISCUSSION TITLE
Making the Email Verification Validation Code length Configurable in Serverpod Auth Module

### DISCUSSION THREAD

**Discussion by adiltirur** [serverpod/serverpod#2221] [Q&A] [answered]
Hello everyone,

I've been working with the Serverpod auth module and I've noticed that the validationCode for email verification is currently set to 8 digits. For the purpose of enhancing user experience in mobile apps, I'm interested in making this length configurable, allowing for either 4 or 6 digit codes.

I'm wondering if there's already a way to configure the length of the validationCode that I might have missed. If not, could anyone provide guidance on how I might implement this feature?

Any suggestions or insights would be greatly appreciated. Thank you in advance for your help.

**Comment by SandPod** [comment]
Hey @adiltirur 👋 

Unfortunately, it is currently not possible to configure the validation code length. But seems lite a reasonable thing to configure so I will open an issue for a feature request so that we can add support for it in the future.

**Reply by Katekko** [reply]
Linking the PR here: https://github.com/serverpod/serverpod/pull/2250
And the issue: https://github.com/serverpod/serverpod/issues/2247

**Reply by SandPod** [reply, marked as answer]
Thanks to @BenAuerDev this has now been implemented in the framework and will be part of the Serverpod 2.0 release.

A new property in the AuthConfig named `validationCodeLength` was introduced to make this possible to configure.

**Reply by adiltirur** [reply]
That is awesome :-) 
Thanks a ton
